### ISSUE TITLE
Feature: Display open applications

### ISSUE THREAD

**Issue by ghost** [acheronfail/apptivator#27]
Would be useful if there was a display of current open applications with their set keyboard shortcuts next to them, you can see how Apptivate App implements it -[apptivateapp.com]( http://www.apptivateapp.com) (first screen shot + many more images / downloadable app on their website for inspiration...)
 
Another example is the discontinued "Launch-it" seen in the second screen shot.

Apptivate
![main_window](https://user-images.githubusercontent.com/21975869/38457896-4ffe3602-3a4b-11e8-8f27-76f36a8af394.png)

Launch-It
![36882_scr](https://user-images.githubusercontent.com/21975869/38457898-568db90c-3a4b-11e8-8b7d-0919d0821e7c.jpg)

**Comment by acheronfail** [comment]
I'm not exactly sure what you mean - if you open the panel ("configure shortcuts") there's the list of added apps, and if you click the add button there's a list of open applications to choose from?

Do you mean a separate dropdown menu that only shows active shortcuts/apps?

**Comment by ghost** [comment]
My mistake, I seem to have blurred the lines between "Application Switcher" and "Application Launcher". I was basically suggesting a view that displays currently open / running applications. I typically have several apps open at a time, and imagined a view that displayed those currently open apps allowing the user to click on one or enter the short cut. In my confusion I pictured this looking similar to the images above. 

So basically, the main menu would display a list of currently running apps that the user can click to switch. The list of added apps with shortcuts would be in a different preference view. Admittedly this is a confused idea, that goes beyond the current scope of this project, but may be a neat feature.

**Comment by acheronfail** [comment]
That's not a bad idea, I've added a similar functionality in 1.4.1 (0ecee12), let me know how that works. 🙂
